recipe/meta.yaml

```diff
@@ -10,19 +10,27 @@ source:
   fn: Healpix_{{ version }}_2016Aug26.tar.gz
   url: https://downloads.sourceforge.net/project/healpix/Healpix_{{ version }}/Healpix_{{ version }}_2016Aug26.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    # backport https://sourceforge.net/p/healpix/code/1189/
+    - 1189.patch
+    # backport https://sourceforge.net/p/healpix/code/1190/
+    - 1190.patch
 
 build:
-  number: 1003
+  number: 1004
   skip: true  # [win]
 
 requirements:
   build:
     - autoconf
-    - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - make
   host:
     - cfitsio
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
```

Comment:
Pls use this for openmp: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=openmp#openmp

Reply:
This has been addressed in the latest force push, thanks @beckermr.